src/components/home/What.jsx

```diff
@@ -5,7 +5,7 @@ const What = () => {
   return (
     <div className="flex flex-col items-center justify-center my-[5%]">
       <Title text="What We Do" />
-      <p className="text-center md:text-2xl text-xl my-3 w-3/4">
+      <p className="text-center text-2xl my-5 w-[90%]">
```

Comment:
revert back the original lines of code and only change margining. you might have forgotten to pull from development branch